enterprise/backend/src/metabase_enterprise/audit_app/api/user.clj

```diff
@@ -18,7 +19,7 @@
   ;; Alerts/DashboardSubscriptions
   (db/delete! PulseChannelRecipient :user_id id)
   ;; archive anything they created.
-  (db/update-where! Pulse {:creator_id id, :archived false} :archived true)
+  (t2/update! Pulse {:creator_id id, :archived false} {:archived true})
```

Comment:
probably safe, `pre-update` on pulse is just doing some assertions on the update values.